src/components/organisms/FilterBar.tsx

```diff
@@ -37,10 +42,22 @@ const FilterBar: FC<IProps> = () => {
             <Grid item className={classes.locationMeta}>
               <Text type={TextType.CONTENT_TITLE}>{store.newsFlashWidgetsMetaString}</Text>
             </Grid>
+            <Grid item className={classes.showDescriptionButton}>
+              <Button.Standard onClick={() => setIsDescOpen(!isDescOpen)}>
+                {isDescOpen ? 'הסתר פרטים' : 'הצג פרטים'}
+              </Button.Standard>
+            </Grid>
           </Grid>
         </Toolbar>
       </AppBar>
       <Divider />
+      {isDescOpen && (
+        <Typography>
+          {store.newsFlashCollection.map((news) => {
```

Comment:
please replace `Typography` with one of our own text elements (see `atoms\Text.tsx` )